specification/servicefabric/data-plane/Microsoft.ServiceFabric/stable/9.0/servicefabric.json

```diff
@@ -28057,6 +28234,38 @@
         ]
       }
     },
+    "ApiVersion_9-0_RequiredQueryParam": {
```

Comment:
What is the story behind this? If exposed in an SDK as a model it will translate into a very bizarre name. For example, I can see the Python name looking something like `ApiVersion90RequiredQueryParam`.  You can set `x-ms-client-name`, but if you are doing that you may as well just pick a better name here.

Reply:
This has been the naming convention SF has used for every api version.